easybuild/easyconfigs/o/OpenCoarrays/OpenCoarrays-1.9.0-gompi-2017a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenCoarrays'
+version = '1.9.0'
+
+homepage = 'https://github.com/sourceryinstitute/opencoarrays'
+description = """OpenCoarrays is an open-source software project that supports
+the coarray Fortran (CAF) parallel programming features of the Fortran 2008
+standard and several features proposed for Fortran 2015 in the draft Technical
+Specification TS 18508 Additional Parallel Features in Fortran."""
+
+toolchain = {'name': 'gompi', 'version': '2017a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['%s/releases/download/%s' % (homepage, version)]
```

Comment:
minor style remark: I'd prefer making this more explicit, and using the `%(version)s` template:

```python
source_urls = ['https://github.com/sourceryinstitute/opencoarrays/releases/download/%(version)s']